src/olympia/versions/tests/test_models.py

```diff
@@ -2076,6 +2076,13 @@ def test_inherit_needs_human_review_with_due_date(self):
             == NeedsHumanReview.REASON_INHERITANCE
         )
 
+        activity_log = (
+            ActivityLog.objects.for_versions(upload_version)
+            .filter(action=amo.LOG.NEEDS_HUMAN_REVIEW_AUTOMATIC.id)
+            .first()
+        )
+        assert activity_log.user == get_task_user()
```

Comment:
mmm, but we fall back to the task user anyway if `core.get_user()` is falsey - which it often is in tests.  So can you either assert `core.get_user()` is non-falsey, or set it if needed?